### ISSUE TITLE
secret in the repo

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/java-docs-samples#3652]
This blocks git commits

elharo@elharo1:~/java-docs-samples/automl/cloud-client$ commit "merge master"
iap/src/test/java/com/example/iap/BuildAndVerifyIapRequestIT.java:45:      "320431926067-ldm6839p8l2sei41nlsfc632l4d0v2u1.apps.googleusercontent.com";

[ERROR] Matched one or more prohibited patterns

**Comment by lesv** [comment]
@elharo Your saying that Github now has something that notices a service account being explicitly checked in?
(or was it our linter?)

I don't think that's a secret of any kind as it's not checked in with a SA key.

What's giving you the message?

What are you asking us to fix / change?

**Comment by elharo** [comment]
This happened in the CLI on my gLinux laptop when I tried to `git commit` my changes (which did not include this file). Never seen this message before.

As you say, it's probably not a problem but in that case we should figure out where this check comes from and turn it off.

**Comment by elharo** [comment]
see https://github.com/awslabs/git-secrets

I'm pretty sure I didn't install this myself but perhaps the gLinux maintainers did?

**Comment by lesv** [comment]
It's not installed on my Mac, nor are there any commit hooks in my version of J-D-S -- I looked. `.git/hooks` - just lots of samples.

I'm going to close this as it only applies to gLinux users.  Feel free to reopen if you disagree.

**Comment by elharo** [comment]
I am going to reopen this since it's not resolved.

I have established that yes, gLinux does install git secrets by default even though gMac doesn't so it's an issue for many developers here.

Two possibilities:

1. This should be a secret and it shouldn't be in the repo. I know in CT4E we went through some gyrations to avoid checking these things into our repo.

2. It's not a secret and we need to somehow configure this repo so it isn't recognized as such.
